<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificationContact</class>
 <widget class="QDialog" name="ModificationContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="LabelNom">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="NomContact"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="LabelPrenom">
      <property name="text">
       <string>PÃ©nom</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="PrenomContact"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="LabelEntreprise">
      <property name="text">
       <string>Entreprise</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="EntrepriseContact"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="LabelEmail">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="EmailContact"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="LabelPhoto">
      <property name="text">
       <string>Uri Photo</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="PhotoContact">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="LabelTelephone">
      <property name="text">
       <string>Telephone</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="TelephoneContact"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 27pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>Modifier Contact</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnnulerModificationContact">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmerModificationContact">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier le contact</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changementPhoto">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>272</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Changer la photo du contact</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
